lib/Service/FavoritesService.php

```diff
@@ -48,7 +48,7 @@ public function isFavorite(int $nodeType, int $id): bool {
 
 			$result = $qb->executeQuery();
 			while ($row = $result->fetch()) {
-				$this->cache->set($cacheKey, true);
+				$this->cache->set(sprintf('%d_%d', $row['node_type'], $row['node_id']), true);
```

Comment:
Good catch, just cacheKey is unsed now ;)

Reply:
Negative, returned in line 55 related to the method parameters.

Reply:
Right, sorry seems I mixed up closing brackets and missed that part